packages/create-plugin/templates/common/.config/webpack/webpack.config.ts

```diff
@@ -147,6 +171,12 @@ const config = async (env): Promise<Configuration> => {
     },
 
     plugins: [
+      // Insert create plugin version information into the bundle
+      new BannerPlugin({
+        banner: "/* [create-plugin] version: " + cpVersion + "*/",
```

Comment:
Very small nit:

I tested this locally and got the following:
```js
/* [create-plugin] version: 4.13.0-canary.970.deb7892.0*/
```

Maybe it would be better to add a space between the version and `*/`, like so?

```js
/* [create-plugin] version: 4.13.0-canary.970.deb7892.0 */
```

```suggestion
        banner: "/* [create-plugin] version: " + cpVersion + " */",
```

Reply:
Ah great spot! Thanks @xnyo. 👍